RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -398,6 +398,11 @@ void equipStatgainIncreasers(boolean[stat] increaseThisStat, boolean alwaysEquip
 			maximizerStatement += "1handed,";		//ignore incompatible weapons
 		}
 	}
+	if(in_bhy())
+	{
+		// can't use garbage tote and wad of used tape gives +MP
+		maximizerStatement += "-equip wad of used tape";
+	}
 	if(!maximize(maximizerStatement,true))
```

Comment:
Fixes a crash in BHY when the maximizer wants to use wad of used tape for whatever reason but you can't use the garbage tote.